training_program/views.py

```diff
@@ -1,22 +1,26 @@
 '''Provide API views for training_program module.'''
-from rest_framework import viewsets
+from rest_framework import viewsets, status, decorators
+from rest_framework.response import Response
 import training_program.models
 import training_program.serializers
 
 
-class ProgramCategoryViewSet(viewsets.ModelViewSet):
-    '''Create API views for ProgramCategory.'''
-    queryset = training_program.models.ProgramCategory.objects.all()
-    serializer_class = training_program.serializers.ProgramCategorySerializer
-
-
-class ProgramFormViewSet(viewsets.ModelViewSet):
-    '''Create API views for ProgarmForm.'''
-    queryset = training_program.models.ProgramForm.objects.all()
-    serializer_class = training_program.serializers.ProgramFormSerializer
-
-
 class ProgramViewSet(viewsets.ModelViewSet):
     '''Create API views for Progarm.'''
     queryset = training_program.models.Program.objects.all()
     serializer_class = training_program.serializers.ProgramSerializer
+
+
+class ProgramCategoryViewSet(viewsets.ViewSet):
+    '''get program categories from background.'''
+    @decorators.action(detail=False, methods=['GET'],
+                       url_path='')
+    def get(self, request):
+        '''return the program categories which are called '''
```

Comment:
Come up with another meaningful comment to describe what this function is doing.